### PR TITLE
Add DTS support for winc1500

### PR DESCRIPTION
**Pull request by galak**
Added support for using DTS for the winc1500 driver.

### CODE REVIEW

**Comment by galak** [comment]
@tbursztyka wanted to push this for your comments, still working on it.

**Comment by galak** [comment]
@tbursztyka can you review now, should be complete.

File: boards/x86/quark_se_c1000_devboard/dts_fixup.h [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/*
```
Comment (nashif) [inline boards/x86/quark_se_c1000_devboard/dts_fixup.h]: why do we have this as part of the board dts fixup? the board does not have this module, it is an add-on that can be added to ANY board.

**Review by nashif — CHANGES_REQUESTED** [review]

File: boards/x86/quark_se_c1000_devboard/dts_fixup.h (hunk 1 above)
Comment (galak) [reply boards/x86/quark_se_c1000_devboard/dts_fixup.h]: Oops, I should move this down into the sample :)

**Review by tbursztyka — APPROVED** [review]

**Comment by galak** [comment]
@nashif take another look when you have a chance

**Review by nashif — APPROVED** [review]

**Pull request merged**